docs/bugs.json

```diff
@@ -193,6 +213,7 @@
         "severity": "very low"
     },
     {
+        "uid": "SOL-2019-7",
         "name": "UninitializedFunctionPointerInConstructor_0.4.x",
```

Comment:
How do we deal with these?
a) Have a unique number as now, 
b) Some alternate scheme like `SOL-2019-7a` and `SOL-2019-7b`,
c) Or since these are the same bugs just for different compilers, we should actually have the same name and uid for them.

I think c) is the right one.

Reply:
Is it split like that to make it possible to release 0.4.27 without this bug?

From a practical standpoint, I think these IDs will mostly be used to look up and compare vulnerabilties so having the same ID for the same bug sounds sensible as long as it is indeed the same bug. For example Etherscan currently displays them like this:

![opensea-contract-compiler-vulnerabilities](https://user-images.githubusercontent.com/137030/115933877-205ace80-a490-11eb-8eaa-706dacde996b.png)

It would make total sense to see `UninitializedFunctionPointerInConstructor` rather than `UninitializedFunctionPointerInConstructor_0.4.x` there. If they used these numeric IDs it would be better if they had the same number for both.

It also makes it easier to track the same vunlerability across multiple compiler versions. The fact that `UninitializedFunctionPointerInConstructor_0.4.x` is no longer reported in 0.5.x might lead some tools to indicate that it's been fixed there and that a new bug was introduced instead :). With naming like a) it would not be immediately apparent that they are the same thing.

Finally, I see in #8725 that github lets you just list individual versions so c) would let us have just one entry there covering both while a) or b) would require two separate entries.